### DISCUSSION TITLE
Welcome to HubChat Discussions!

### DISCUSSION THREAD

**Discussion by williamanjo** [Maker-PI/HubChat#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by williamanjo** [comment]
@StarSpangledBanner @ubiratanlsc @vitorjf07  @Flipjone <br>
Se precisarem de ajuda em algo , ou tiverem uma ideia , Por favor, Enviar nesta discussão

**Reply by Flipjone** [reply]
@ubiratanlsc 
Sera que o Ubiratan vai ler isso algum dia? kkkkk

Devido ao pedido do Scrum Master, eu queria que você mudasse um pouco aquele ícone do HubChat que você fez.

![HubChat Icon](https://user-images.githubusercontent.com/70604266/121985547-523f3180-cd6b-11eb-85f9-22acac3ce1ca.jpg)

Para ser especifico, aqueles 3 pontinhos que ficaram no rosto do Mascote estão parecendo olhos; uma solução facil seria apenas esticar os pontinhos para que se tornem pequenas linhas, mas você quem sabe.

**Reply by ubiratanlsc** [reply]
Talvez ele nunca leia
